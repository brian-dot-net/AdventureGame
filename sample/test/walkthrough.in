hello
look

heeyyyyyyyy
HI
take care
look table
GET KEY
look sun
move table
move out
move table
look
look coin
get table
inv
read coin
GET COIN
read coin
inv
drop
THROW coin
read coin
take coin
look coin
exit
-- END --