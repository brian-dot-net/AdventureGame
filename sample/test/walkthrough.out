You are in the main room.
There is a table here.
> You say, "Hello," to no one in particular. No one answers.
> You are in the main room.
There is a table here.
> > You can't do that.
> You say, "Hello," to no one in particular. No one answers.
> You can't take that.
> It is an ordinary wooden table.
> You can't GET that.
> You see nothing of interest.
> You move the table slightly. Underneath you see a coin.
> You can't do that.
> Someone has already moved it.
> You are in the main room.
There is a table here.
There is a coin here.
> It is a small gold coin with an inscription on the edge.
> 